psg/src/main.c

```diff
@@ -0,0 +1,78 @@
+// md-toolchain example main.c
+// Michael Moffitt 2018
+//
+// This main shows a simple "hello world" demo.
```

Comment:
This comment block is a leftover from when this started its life as hello world.

Reply:
PR updated